### DISCUSSION TITLE
HRR Evolution

### DISCUSSION THREAD

**Discussion by SXFire12** [firemodels/cfast#2111] [Q&A]
Dear All, 

I tried to compare a very simple model between FDS & CFAST. The idea is to compare the actual HRR. After the simulation, both FDS and CFAST yielded HRR less than what I expected because of insufficient oxygen availability. However, I calculated manually, and results from FDS are quite reasonable rather to the CFAST. What are the probable reasons to explain that?

I attached my files as well. 

Thank you
[HRR.zip](https://github.com/user-attachments/files/15516299/HRR.zip)

**Comment by mcgratta** [comment]
Hard to say without running the cases, but my guess is that CFAST cuts off the fire when the smoke layer drops down to its base. Also, CFAST uses a fixed value of oxygen concentration to stop the combustion. FDS uses O2 and temperature for extinction criteria.

**Reply by SXFire12** [reply]
Thanks for the prompt reply, Kevin. Could you have a look at the attachment file for further suggestions from your side? I ran for both of the files took me less than 5 minutes.
